### PR TITLE
GMS-975: Update boilerplate contract and scripts to reflect renamed contracts.

### PR DESCRIPTION
**Pull request by dlevy47**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,21 +8,35 @@
...
     "deploy:goerli": "npx hardhat run scripts/deploy.ts --network goerli",
-    "deploy:zkevm": "npx hardhat run scripts/deploy.ts --network immutableZkevmTestnet"
+    "deploy:zkevm": "npx hardhat run scripts/deploy.ts --network immutableZkevmTestnet",
+    "deploy:by-id:zkevm": "npx hardhat run scripts/by-id/deploy.ts --network immutableZkevmTestnet"
   },
   "author": "Immutable",
   "license": "MIT",
   "dependencies": {
-    "@imtbl/zkevm-contracts": "^1.0.5",
+    "@imtbl/sdk": "latest",
```
Comment (ipekt) [inline package.json:17]: is this needed?

Comment (dlevy47) [reply package.json]: Oops, nope. Fixed.

File: package.json [hunk 2]
```diff
@@ -8,21 +8,34 @@
...
     "deploy:goerli": "npx hardhat run scripts/deploy.ts --network goerli",
-    "deploy:zkevm": "npx hardhat run scripts/deploy.ts --network immutableZkevmTestnet"
+    "deploy:zkevm": "npx hardhat run scripts/deploy.ts --network immutableZkevmTestnet",
+    "deploy:by-id:zkevm": "npx hardhat run scripts/by-id/deploy.ts --network immutableZkevmTestnet"
   },
   "author": "Immutable",
   "license": "MIT",
   "dependencies": {
-    "@imtbl/zkevm-contracts": "^1.0.5",
+    "@imtbl/zkevm-contracts": "^1.0.9",
```
Comment (ipekt) [inline package.json:17]: could we update this to latest?

**Review by ipekt — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (dlevy47) [reply package.json]: Done.

**Review by ipekt — APPROVED** [review]

**Pull request merged**